cabal-helper.cabal

```diff
@@ -56,16 +56,22 @@ library
                        Paths_cabal_helper
   ghc-options:         -Wall
   build-depends:       base          < 5    && >= 4.5
-                     , Cabal         < 2.1  && >= 2.0     || < 1.26 && >= 1.14
+  if os(windows)
+    build-depends:     base                 && >= 4.7
```

Comment:
This doesn't parse at all, did you even build before pushing!?

Reply:
My bad, i'm terribly sorry.